Compile Fails due to Faulty pipewire Version Check

<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
<!--- Read the following link before you create a new problem report: https://kodi.wiki/view/HOW-TO:Submit_a_bug_report  -->
## Bug report
### Describe the bug
Here is a clear and concise description of what the problem is:
Using the compile instructions for Kodi in CentOS 9.1 the compile crashed due to a faulty version check of pipewire.  Hasn't anyone noticed this?  It doesn't compile.

<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
    [ 13%] Building CXX object build/cores/paplayer/CMakeFiles/paplayer.dir/AudioDecoder.cpp.o
    cd /home/carl/src/kodi-build/build/cores/paplayer && /usr/bin/ccache /usr/lib64/ccache/c++  -I/home/carl/src/kodi_21.0 -I/home/carl/src/kodi_21.0/lib -I/home/carl/src/kodi_21.0/xbmc -I/home/carl/src/kodi_21.0/xbmc/platform/linux -I/home/carl/src/kodi_21.0/xbmc/cores/VideoPlayer -I/home/carl/src/kodi-build/build -I/home/carl/src/kodi_21.0/xbmc/platform/posix -isystem /home/carl/src/kodi-build/build/include -isystem /usr/include/dbus-1.0 -isystem /usr/lib64/dbus-1.0/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -isystem /usr/include/python3.9 -isystem /usr/include/samba-4.0 -isystem /usr/include/libxml2 -isystem /usr/include/ffmpeg -isystem /home/carl/src/kodi-build/build/cores/RetroPlayer/messages -isystem /usr/include/freetype2 -isystem /usr/include/fribidi -isystem /home/carl/src/kodi_21.0/xbmc/contrib -isystem /usr/include/lzo -isystem /usr/include/libdrm -Wall -Wdouble-promotion -Wmissing-field-initializers -Wsign-compare -Wextra -Wno-unused-parameter -Wnon-virtual-dtor -O3 -DNDEBUG -DTARGET_POSIX -DTARGET_LINUX -D_GNU_SOURCE -DHAVE_LINUX_UDMABUF=1 -DHAVE_LINUX_DMA_HEAP=1 -DHAVE_LINUX_DMA_BUF=1 -DHAVE_MKOSTEMP=1 -DHAVE_LINUX_MEMFD=1 -DHAVE_STATX=1 -DHAVE_SSE=1 -DHAVE_SSE2=1 -DHAVE_SSE3=1 -DHAVE_SSSE3=1 -DHAVE_SSE4_1=1 -D__STDC_CONSTANT_MACROS -D_FILE_OFFSET_BITS=64 -DHAS_POSIX_NETWORK -DHAS_LINUX_NETWORK -DHAS_BUILTIN_SYNC_ADD_AND_FETCH=1 -DHAS_BUILTIN_SYNC_SUB_AND_FETCH=1 -DHAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP=1 -DHAVE_INOTIFY=1 -DHAVE_POSIX_FADVISE=1 -DHAVE_LOCALTIME_R=1 -DHAVE_GMTIME_R=1 -DHAVE_INTTYPES_H=1 -DHAS_ALSA=1 -DHAS_AVAHI=1 -DHAS_ZEROCONF=1 -DHAVE_LIBBLURAY=1 -DHAVE_LIBBLURAY_BDJ=1 -DHAVE_LIBCEC=1 -DHAS_DBUS=1 -DHAS_ISO9660PP=1 -DHAVE_LCMS2=1 -DCMS_NO_REGISTER_KEYWORD=1 -DHAS_LIRC=1 -DHAS_WEB_SERVER=1 -DHAS_WEB_INTERFACE=1 -DHAS_PIPEWIRE=1 -DHAS_AIRPLAY=1 -DHAS_PULSEAUDIO=1 -DHAS_PYTHON=1 -DHAS_FILESYSTEM_SMB=1 -DHAVE_LIBUDEV=1 -DHAS_UDFREAD=1 -DHAVE_LIBXSLT=1 -DHAVE_LIBVA=1 -DHAS_GLX=1 -DHAVE_LIBVDPAU=1 -DHAVE_NEW_CROSSGUID -DFFMPEG_VER_SHA=\"5.0.0\" -DSPDLOG_FMT_EXTERNAL -DSPDLOG_DEBUG_ON -DSPDLOG_NO_ATOMIC_LEVELS -DSPDLOG_ENABLE_PATTERN_PADDING -DSPDLOG_COMPILED_LIB -DHAS_EGL=1 -DHAVE_EGLEXTANGLE=1 -DHAVE_X11=1 -DHAVE_LIBXRANDR=1 -DHAVE_HDR_OUTPUT_METADATA=1 -DHAVE_DRM_MODIFIER_NAME=1 -DHAS_GL=1 -DHAS_MYSQL=1 -DHAS_UPNP=1 -DHAS_DVD_DRIVE -DHAS_CDDA_RIPPER -DBIN_INSTALL_PATH=\"/usr/local/lib64/kodi\" -DINSTALL_PATH=\"/usr/local/share/kodi\" -Werror=double-promotion -Werror=missing-field-initializers -Werror=sign-compare -std=c++17 -MD -MT build/cores/paplayer/CMakeFiles/paplayer.dir/AudioDecoder.cpp.o -MF CMakeFiles/paplayer.dir/AudioDecoder.cpp.o.d -o CMakeFiles/paplayer.dir/AudioDecoder.cpp.o -c /home/carl/src/kodi_21.0/xbmc/cores/paplayer/AudioDecoder.cpp
    /home/carl/src/kodi_21.0/xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp: In function ‘std::chrono::duration<double, std::ratio<1> > {anonymous}::PWTimeToAEDelay(const pw_time&, const uint32_t&)’:
    /home/carl/src/kodi_21.0/xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp:235:13: error: ‘const struct pw_time’ has no member named ‘buffered’
      235 |       (time.buffered * fraction) + ((time.delay - elapsed) * fraction) +
          |             ^~~~~~~~
    gmake[2]: *** [build/cores/audioengine/CMakeFiles/audioengine.dir/build.make:370: build/cores/audioengine/CMakeFiles/audioengine.dir/Sinks/pipewire/AESinkPipewire.cpp.o] Error 1
    gmake[2]: Leaving directory '/home/carl/src/kodi-build'
    gmake[1]: *** [CMakeFiles/Makefile2:6341: build/cores/audioengine/CMakeFiles/audioengine.dir/all] Error 2
    gmake[1]: *** Waiting for unfinished jobs....
     
    [ 13%] Building CXX object build/cores/paplayer/CMakeFiles/paplayer.dir/CodecFactory.cpp.o
    cd /home/carl/src/kodi-build/build/cores/paplayer && /usr/bin/ccache /usr/lib64/ccache/c++  -I/home/carl/src/kodi_21.0 -I/home/carl/src/kodi_21.0/lib -I/home/carl/src/kodi_21.0/xbmc -I/home/carl/src/kodi_21.0/xbmc/platform/linux -I/home/carl/src/kodi_21.0/xbmc/cores/VideoPlayer -I/home/carl/src/kodi-build/build -I/home/carl/src/kodi_21.0/xbmc/platform/posix -isystem /home/carl/src/kodi-build/build/include -isystem /usr/include/dbus-1.0 -isystem /usr/lib64/dbus-1.0/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -isystem /usr/include/python3.9 -isystem /usr/include/samba-4.0 -isystem /usr/include/libxml2 -isystem /usr/include/ffmpeg -isystem /home/carl/src/kodi-build/build/cores/RetroPlayer/messages -isystem /usr/include/freetype2 -isystem /usr/include/fribidi -isystem /home/carl/src/kodi_21.0/xbmc/contrib -isystem /usr/include/lzo -isystem /usr/include/libdrm -Wall -Wdouble-promotion -Wmissing-field-initializers -Wsign-compare -Wextra -Wno-unused-parameter -Wnon-virtual-dtor -O3 -DNDEBUG -DTARGET_POSIX -DTARGET_LINUX -D_GNU_SOURCE -DHAVE_LINUX_UDMABUF=1 -DHAVE_LINUX_DMA_HEAP=1 -DHAVE_LINUX_DMA_BUF=1 -DHAVE_MKOSTEMP=1 -DHAVE_LINUX_MEMFD=1 -DHAVE_STATX=1 -DHAVE_SSE=1 -DHAVE_SSE2=1 -DHAVE_SSE3=1 -DHAVE_SSSE3=1 -DHAVE_SSE4_1=1 -D__STDC_CONSTANT_MACROS -D_FILE_OFFSET_BITS=64 -DHAS_POSIX_NETWORK -DHAS_LINUX_NETWORK -DHAS_BUILTIN_SYNC_ADD_AND_FETCH=1 -DHAS_BUILTIN_SYNC_SUB_AND_FETCH=1 -DHAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP=1 -DHAVE_INOTIFY=1 -DHAVE_POSIX_FADVISE=1 -DHAVE_LOCALTIME_R=1 -DHAVE_GMTIME_R=1 -DHAVE_INTTYPES_H=1 -DHAS_ALSA=1 -DHAS_AVAHI=1 -DHAS_ZEROCONF=1 -DHAVE_LIBBLURAY=1 -DHAVE_LIBBLURAY_BDJ=1 -DHAVE_LIBCEC=1 -DHAS_DBUS=1 -DHAS_ISO9660PP=1 -DHAVE_LCMS2=1 -DCMS_NO_REGISTER_KEYWORD=1 -DHAS_LIRC=1 -DHAS_WEB_SERVER=1 -DHAS_WEB_INTERFACE=1 -DHAS_PIPEWIRE=1 -DHAS_AIRPLAY=1 -DHAS_PULSEAUDIO=1 -DHAS_PYTHON=1 -DHAS_FILESYSTEM_SMB=1 -DHAVE_LIBUDEV=1 -DHAS_UDFREAD=1 -DHAVE_LIBXSLT=1 -DHAVE_LIBVA=1 -DHAS_GLX=1 -DHAVE_LIBVDPAU=1 -DHAVE_NEW_CROSSGUID -DFFMPEG_VER_SHA=\"5.0.0\" -DSPDLOG_FMT_EXTERNAL -DSPDLOG_DEBUG_ON -DSPDLOG_NO_ATOMIC_LEVELS -DSPDLOG_ENABLE_PATTERN_PADDING -DSPDLOG_COMPILED_LIB -DHAS_EGL=1 -DHAVE_EGLEXTANGLE=1 -DHAVE_X11=1 -DHAVE_LIBXRANDR=1 -DHAVE_HDR_OUTPUT_METADATA=1 -DHAVE_DRM_MODIFIER_NAME=1 -DHAS_GL=1 -DHAS_MYSQL=1 -DHAS_UPNP=1 -DHAS_DVD_DRIVE -DHAS_CDDA_RIPPER -DBIN_INSTALL_PATH=\"/usr/local/lib64/kodi\" -DINSTALL_PATH=\"/usr/local/share/kodi\" -Werror=double-promotion -Werror=missing-field-initializers -Werror=sign-compare -std=c++17 -MD -MT build/cores/paplayer/CMakeFiles/paplayer.dir/CodecFactory.cpp.o -MF CMakeFiles/paplayer.dir/CodecFactory.cpp.o.d -o CMakeFiles/paplayer.dir/CodecFactory.cpp.o -c /home/carl/src/kodi_21.0/xbmc/cores/paplayer/CodecFactory.cpp
     
    [ 13%] Building CXX object build/cores/paplayer/CMakeFiles/paplayer.dir/PAPlayer.cpp.o
    cd /home/carl/src/kodi-build/build/cores/paplayer && /usr/bin/ccache /usr/lib64/ccache/c++  -I/home/carl/src/kodi_21.0 -I/home/carl/src/kodi_21.0/lib -I/home/carl/src/kodi_21.0/xbmc -I/home/carl/src/kodi_21.0/xbmc/platform/linux -I/home/carl/src/kodi_21.0/xbmc/cores/VideoPlayer -I/home/carl/src/kodi-build/build -I/home/carl/src/kodi_21.0/xbmc/platform/posix -isystem /home/carl/src/kodi-build/build/include -isystem /usr/include/dbus-1.0 -isystem /usr/lib64/dbus-1.0/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -isystem /usr/include/python3.9 -isystem /usr/include/samba-4.0 -isystem /usr/include/libxml2 -isystem /usr/include/ffmpeg -isystem /home/carl/src/kodi-build/build/cores/RetroPlayer/messages -isystem /usr/include/freetype2 -isystem /usr/include/fribidi -isystem /home/carl/src/kodi_21.0/xbmc/contrib -isystem /usr/include/lzo -isystem /usr/include/libdrm -Wall -Wdouble-promotion -Wmissing-field-initializers -Wsign-compare -Wextra -Wno-unused-parameter -Wnon-virtual-dtor -O3 -DNDEBUG -DTARGET_POSIX -DTARGET_LINUX -D_GNU_SOURCE -DHAVE_LINUX_UDMABUF=1 -DHAVE_LINUX_DMA_HEAP=1 -DHAVE_LINUX_DMA_BUF=1 -DHAVE_MKOSTEMP=1 -DHAVE_LINUX_MEMFD=1 -DHAVE_STATX=1 -DHAVE_SSE=1 -DHAVE_SSE2=1 -DHAVE_SSE3=1 -DHAVE_SSSE3=1 -DHAVE_SSE4_1=1 -D__STDC_CONSTANT_MACROS -D_FILE_OFFSET_BITS=64 -DHAS_POSIX_NETWORK -DHAS_LINUX_NETWORK -DHAS_BUILTIN_SYNC_ADD_AND_FETCH=1 -DHAS_BUILTIN_SYNC_SUB_AND_FETCH=1 -DHAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP=1 -DHAVE_INOTIFY=1 -DHAVE_POSIX_FADVISE=1 -DHAVE_LOCALTIME_R=1 -DHAVE_GMTIME_R=1 -DHAVE_INTTYPES_H=1 -DHAS_ALSA=1 -DHAS_AVAHI=1 -DHAS_ZEROCONF=1 -DHAVE_LIBBLURAY=1 -DHAVE_LIBBLURAY_BDJ=1 -DHAVE_LIBCEC=1 -DHAS_DBUS=1 -DHAS_ISO9660PP=1 -DHAVE_LCMS2=1 -DCMS_NO_REGISTER_KEYWORD=1 -DHAS_LIRC=1 -DHAS_WEB_SERVER=1 -DHAS_WEB_INTERFACE=1 -DHAS_PIPEWIRE=1 -DHAS_AIRPLAY=1 -DHAS_PULSEAUDIO=1 -DHAS_PYTHON=1 -DHAS_FILESYSTEM_SMB=1 -DHAVE_LIBUDEV=1 -DHAS_UDFREAD=1 -DHAVE_LIBXSLT=1 -DHAVE_LIBVA=1 -DHAS_GLX=1 -DHAVE_LIBVDPAU=1 -DHAVE_NEW_CROSSGUID -DFFMPEG_VER_SHA=\"5.0.0\" -DSPDLOG_FMT_EXTERNAL -DSPDLOG_DEBUG_ON -DSPDLOG_NO_ATOMIC_LEVELS -DSPDLOG_ENABLE_PATTERN_PADDING -DSPDLOG_COMPILED_LIB -DHAS_EGL=1 -DHAVE_EGLEXTANGLE=1 -DHAVE_X11=1 -DHAVE_LIBXRANDR=1 -DHAVE_HDR_OUTPUT_METADATA=1 -DHAVE_DRM_MODIFIER_NAME=1 -DHAS_GL=1 -DHAS_MYSQL=1 -DHAS_UPNP=1 -DHAS_DVD_DRIVE -DHAS_CDDA_RIPPER -DBIN_INSTALL_PATH=\"/usr/local/lib64/kodi\" -DINSTALL_PATH=\"/usr/local/share/kodi\" -Werror=double-promotion -Werror=missing-field-initializers -Werror=sign-compare -std=c++17 -MD -MT build/cores/paplayer/CMakeFiles/paplayer.dir/PAPlayer.cpp.o -MF CMakeFiles/paplayer.dir/PAPlayer.cpp.o.d -o CMakeFiles/paplayer.dir/PAPlayer.cpp.o -c /home/carl/src/kodi_21.0/xbmc/cores/paplayer/PAPlayer.cpp
     
    [ 13%] Building CXX object build/cores/paplayer/CMakeFiles/paplayer.dir/VideoPlayerCodec.cpp.o
    cd /home/carl/src/kodi-build/build/cores/paplayer && /usr/bin/ccache /usr/lib64/ccache/c++  -I/home/carl/src/kodi_21.0 -I/home/carl/src/kodi_21.0/lib -I/home/carl/src/kodi_21.0/xbmc -I/home/carl/src/kodi_21.0/xbmc/platform/linux -I/home/carl/src/kodi_21.0/xbmc/cores/VideoPlayer -I/home/carl/src/kodi-build/build -I/home/carl/src/kodi_21.0/xbmc/platform/posix -isystem /home/carl/src/kodi-build/build/include -isystem /usr/include/dbus-1.0 -isystem /usr/lib64/dbus-1.0/include -isystem /usr/include/pipewire-0.3 -isystem /usr/include/spa-0.2 -isystem /usr/include/python3.9 -isystem /usr/include/samba-4.0 -isystem /usr/include/libxml2 -isystem /usr/include/ffmpeg -isystem /home/carl/src/kodi-build/build/cores/RetroPlayer/messages -isystem /usr/include/freetype2 -isystem /usr/include/fribidi -isystem /home/carl/src/kodi_21.0/xbmc/contrib -isystem /usr/include/lzo -isystem /usr/include/libdrm -Wall -Wdouble-promotion -Wmissing-field-initializers -Wsign-compare -Wextra -Wno-unused-parameter -Wnon-virtual-dtor -O3 -DNDEBUG -DTARGET_POSIX -DTARGET_LINUX -D_GNU_SOURCE -DHAVE_LINUX_UDMABUF=1 -DHAVE_LINUX_DMA_HEAP=1 -DHAVE_LINUX_DMA_BUF=1 -DHAVE_MKOSTEMP=1 -DHAVE_LINUX_MEMFD=1 -DHAVE_STATX=1 -DHAVE_SSE=1 -DHAVE_SSE2=1 -DHAVE_SSE3=1 -DHAVE_SSSE3=1 -DHAVE_SSE4_1=1 -D__STDC_CONSTANT_MACROS -D_FILE_OFFSET_BITS=64 -DHAS_POSIX_NETWORK -DHAS_LINUX_NETWORK -DHAS_BUILTIN_SYNC_ADD_AND_FETCH=1 -DHAS_BUILTIN_SYNC_SUB_AND_FETCH=1 -DHAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP=1 -DHAVE_INOTIFY=1 -DHAVE_POSIX_FADVISE=1 -DHAVE_LOCALTIME_R=1 -DHAVE_GMTIME_R=1 -DHAVE_INTTYPES_H=1 -DHAS_ALSA=1 -DHAS_AVAHI=1 -DHAS_ZEROCONF=1 -DHAVE_LIBBLURAY=1 -DHAVE_LIBBLURAY_BDJ=1 -DHAVE_LIBCEC=1 -DHAS_DBUS=1 -DHAS_ISO9660PP=1 -DHAVE_LCMS2=1 -DCMS_NO_REGISTER_KEYWORD=1 -DHAS_LIRC=1 -DHAS_WEB_SERVER=1 -DHAS_WEB_INTERFACE=1 -DHAS_PIPEWIRE=1 -DHAS_AIRPLAY=1 -DHAS_PULSEAUDIO=1 -DHAS_PYTHON=1 -DHAS_FILESYSTEM_SMB=1 -DHAVE_LIBUDEV=1 -DHAS_UDFREAD=1 -DHAVE_LIBXSLT=1 -DHAVE_LIBVA=1 -DHAS_GLX=1 -DHAVE_LIBVDPAU=1 -DHAVE_NEW_CROSSGUID -DFFMPEG_VER_SHA=\"5.0.0\" -DSPDLOG_FMT_EXTERNAL -DSPDLOG_DEBUG_ON -DSPDLOG_NO_ATOMIC_LEVELS -DSPDLOG_ENABLE_PATTERN_PADDING -DSPDLOG_COMPILED_LIB -DHAS_EGL=1 -DHAVE_EGLEXTANGLE=1 -DHAVE_X11=1 -DHAVE_LIBXRANDR=1 -DHAVE_HDR_OUTPUT_METADATA=1 -DHAVE_DRM_MODIFIER_NAME=1 -DHAS_GL=1 -DHAS_MYSQL=1 -DHAS_UPNP=1 -DHAS_DVD_DRIVE -DHAS_CDDA_RIPPER -DBIN_INSTALL_PATH=\"/usr/local/lib64/kodi\" -DINSTALL_PATH=\"/usr/local/share/kodi\" -Werror=double-promotion -Werror=missing-field-initializers -Werror=sign-compare -std=c++17 -MD -MT build/cores/paplayer/CMakeFiles/paplayer.dir/VideoPlayerCodec.cpp.o -MF CMakeFiles/paplayer.dir/VideoPlayerCodec.cpp.o.d -o CMakeFiles/paplayer.dir/VideoPlayerCodec.cpp.o -c /home/carl/src/kodi_21.0/xbmc/cores/paplayer/VideoPlayerCodec.cpp
     
    [ 14%] Linking CXX static library paplayer.a
    cd /home/carl/src/kodi-build/build/cores/paplayer && /usr/bin/cmake -P CMakeFiles/paplayer.dir/cmake_clean_target.cmake
    cd /home/carl/src/kodi-build/build/cores/paplayer && /usr/bin/cmake -E cmake_link_script CMakeFiles/paplayer.dir/link.txt --verbose=1
    /usr/bin/ar qc paplayer.a CMakeFiles/paplayer.dir/AudioDecoder.cpp.o CMakeFiles/paplayer.dir/CodecFactory.cpp.o CMakeFiles/paplayer.dir/PAPlayer.cpp.o CMakeFiles/paplayer.dir/VideoPlayerCodec.cpp.o
    /usr/bin/ranlib paplayer.a
    gmake[2]: Leaving directory '/home/carl/src/kodi-build'
    [ 14%] Built target paplayer
    gmake[1]: Leaving directory '/home/carl/src/kodi-build'
    gmake: *** [Makefile:146: all] Error 2


<!--- Put your text below this line -->
The kodi source seems to depend on pipewire >=0.3.34 and CentOS 9.1 has 0.3.47.
However `buffered` field seems to have been added in 0.3.50 (https://github.com/PipeWire/pipewire/commit/5a9d2679ca480db7083887e9c48db7367e4bb1f5) so xbmc's version check is wrong


## Expected Behavior
Here is a clear and concise description of what was expected to happen:
<!--- Tell us what should happen -->
<!--- Put your text below this line -->
Kodi should compile.


## Actual Behavior
<!--- Tell us what happens instead -->
<!--- Put your text below this line -->
It doesn't compile, and I have no other choice as there is no package for RHEL.


## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->
<!--- Put your text below this line -->
Fix the pipewire version check.


### To Reproduce
Steps to reproduce the behavior:
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
<!--- Put your text below this line -->
1. configure
2. compile
3. fail


### Debuglog
<!--- Put your text below this line -->
<!--- A debuglog is always mandatory when creating an issue. Provide one! -->
The debuglog can be found here:
A debuglog can be found where?


### Your Environment
Used Operating system:
<!--- Include as many relevant details about the environment you experienced the bug in -->
<!--- Put your text below this line. Checkboxes can easily be ticked once issue is created -->
 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [X ] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:  CentOS 9.1
 - Kodi version: 21



<!--- End of this issue -->
*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

--- a/cmake/modules/FindPipewire.cmake
+++ b/cmake/modules/FindPipewire.cmake
@@ -12,7 +12,7 @@
 #
 
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_PIPEWIRE libpipewire-0.3>=0.3.34 QUIET)
+  pkg_check_modules(PC_PIPEWIRE libpipewire-0.3>=0.3.50 QUIET)
   pkg_check_modules(PC_SPA libspa-0.2>=0.2 QUIET)
 endif()
 

